### ISSUE TITLE
gulp dev erroring on watch

### ISSUE THREAD

**Issue by mherodev** [SnowdogApps/magento2-frontools#169]
When running `gulp dev` I am running into the following error:

```
/Users/mherold/git/project-magento/magento/vendor/snowdog/frontools/task/watch.js:88
        require('../helper/css-lint')(gulp, plugins, config, name, event.path)();
                                                                              ^

TypeError: require(...)(...) is not a function
    at gulp.watch.event (/Users/mherold/git/project-magento/magento/vendor/snowdog/frontools/task/watch.js:88:79)
    at Gaze.<anonymous> (/Users/mherold/git/project-magento/magento/vendor/snowdog/frontools/node_modules/glob-watcher/index.js:18:14)
    at emitTwo (events.js:106:13)
    at Gaze.emit (events.js:191:7)
    at Gaze.emit (/Users/mherold/git/project-magento/magento/vendor/snowdog/frontools/node_modules/gaze/lib/gaze.js:129:32)
    at /Users/mherold/git/project-magento/magento/vendor/snowdog/frontools/node_modules/gaze/lib/gaze.js:415:16
    at StatWatcher._pollers.(anonymous function) (/Users/mherold/git/project-magento/magento/vendor/snowdog/frontools/node_modules/gaze/lib/gaze.js:326:7)
    at emitTwo (events.js:106:13)
    at StatWatcher.emit (events.js:191:7)
    at StatWatcher._handle.onchange (fs.js:1487:10)
```

This didn't happen yesterday, so it must be a result of today's release.

In the mean time, is there an easy way to rollback via composer?

Thanks in advance!

**Comment by mherodev** [comment]
Giving `composer require snowdog/frontools:1.3.3` a shot.

**Comment by mherodev** [comment]
That does it! Good in the mean time.

**Comment by Igloczek** [comment]
Sorry m8, I totally forgot to check files watching. Will fix this tomorrow.
As you said, for now let's stick with 1.3.3.

**Comment by Igloczek** [comment]
Sorry m8, I totally forgot to check files watching. Will fix this tomorrow.
As you said, for now let's stick with 1.3.3.

**Comment by Igloczek** [comment]
Fixed in 1.3.6
